io.github.wesenseged.Amharic-bible.json

```
@@ -0,0 +1,27 @@
+{
+	"id": "io.github.wesenseged.Amharic-bible",
+	"runtime": "org.kde.Platform",
+	"runtime-version": "6.10",
+	"sdk": "org.kde.Sdk",
+	"command": "amharic-bible",
+	"finish-args": [
+		"--share=ipc",
+		"--socket=wayland",
+		"--socket=fallback-x11",
+		"--device=dri"
+	],
+	"modules": [
+		{
+			"name": "amharic-bible",
+			"buildsystem": "cmake-ninja",
+			"builddir": true,
+			"sources": [
+				{
+					"type": "git",
+					"url": "https://github.com/wesenseged/Amharic-bible.git",
+					"tag": "0.0.1"
```

```suggestion
					"tag": "0.0.1",
					"commit": "df9ae511aad5cdac7acf3377cade86732b835015"
```

this protect against retagging.